src/Components/Main/MessagesPage/Dialog/Dialog.jsx

```diff
@@ -0,0 +1,12 @@
+import { NavLink } from 'react-router-dom';
+import classes from './Dialog.module.scss';
+import classNames from 'classnames';
```

Comment:
you could simplify the name, it can be just "cn" =)

Reply:
Done!